next.config.js

```
@@ -29,6 +29,17 @@ const {
       typescript: {
         ignoreBuildErrors: true
       },
+      async headers() {
```

ESLint

[error] Expected indentation of 4 spaces but found 6.

(indent)